types/custom-functions-runtime/index.d.ts

```diff
@@ -25,6 +27,24 @@ declare namespace CustomFunctions {
      */
     function associate(mappings: { [key: string]: Function }): void;
 
+    /**
+     * Use this class to handle errors and write custom error messages.
+     * [Api set: CustomFunctionsRuntime 1.2]
+     */
+    class Error {
+        constructor(errorCode: ErrorCode, errorMessage?: string)
```

Comment:
Adding a semicolon at end of line
```suggestion
        constructor(errorCode: ErrorCode, errorMessage?: string);
```